<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberQuery</class>
 <widget class="QWidget" name="memberQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会员查询</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="conditionComboBox">
     <item>
      <property name="text">
       <string>所有会员</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>会员ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>会员姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>联系电话</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>邮箱</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="valueLineEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="queryButton">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>